tx_to_mesh_plugin.py

```
@@ -36,88 +34,87 @@ class Plugin(BasePlugin):
-        else:
-            # Keep the full message including prefix
-            self.logger.debug(f"Keeping full message: '{original_message}'")
-            return original_message
-
-    async def handle_meshtastic_message(self, packet, formatted_message: str,
-                                      longname: str, meshnet_name: str):
+        # This plugin only handles Matrix -> Meshtastic messages, not DMs
+        # Always return False for DMs to ignore direct commands
+        # Matrix messages should be mapped to specific Meshtastic channels
+        if not self.is_channel_enabled(0, is_direct_message=False):
+            self.logger.debug(f"tx_to_mesh: not enabled for this room/channel")
```

Ruff (0.14.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)